### PR TITLE
[ACM-6284]: Restructuring configure_hosted_aws.adoc

### PR DESCRIPTION
**Pull request by lahinson**
Related Jira: https://issues.redhat.com/browse/ACM-6284

This PR restructures `configure_hosted_aws.adoc` to be several shorter files rather than one very long file.

### CODE REVIEW

File: clusters/hosted_control_planes/configure_hosted_bm.adoc [hunk 1]
```diff
@@ -64,3 +64,5 @@ After you meet the prerequisites, complete the following steps to configure host
 . xref:../hosted_control_planes/hosted_create_bare_metal.adoc#creating-a-hosted-cluster-bm[Create a hosted cluster on bare metal]
 . xref:../hosted_control_planes/hosting_service_cluster_access.adoc#access-hosted-cluster[Access the hosted cluster]
 . xref:../hosted_control_planes/verifying_hosted_cluster.adoc#verifying-cluster-creation[Verify hosted cluster creation]
+. xref:../hosted_control_planes/hosted_install_cli.adoc#hosted-install-cli[Installing the hosted control planes command line interface (hcp)]
+//lahinson - sept. 2023 - should the hcp cli topic be listed as a prerequisite in this file?
```
Comment (lahinson) [inline clusters/hosted_control_planes/configure_hosted_bm.adoc]: @rokej When you can, let me know your thoughts about the placement of the hcp cli topic in this file. Thanks!

File: clusters/hosted_control_planes/hosted_install_cli.adoc [hunk 2]
```diff
@@ -0,0 +1,38 @@
+[#hosted-install-cli]
+= Installing the hosted control planes command line interface
+
+The hosted control planes (`hypershift`) command line interface is used to create and manage {ocp-short} hosted control plane clusters. After you enable the hosted control planes feature, you can install the hosted control planes command line interface by completing the following steps:
```
Comment (rokej) [inline clusters/hosted_control_planes/hosted_install_cli.adoc]: Rename `hypershift` to `hcp`

File: clusters/hosted_control_planes/hosted_install_cli.adoc [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+. From the {ocp-short} console, click the *Help icon* > *Command Line Tools*.
+
+. Click *Download hypershift CLI* for your platform.
```
Comment (rokej) [inline clusters/hosted_control_planes/hosted_install_cli.adoc]: Download hcp CLI

File: clusters/hosted_control_planes/hosted_install_cli.adoc [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+
+*Note:* The download is only visible if you have enabled the `hypershift-preview` feature.
+
+. Unpack the downloaded archive by running the following command:
+
++
+----
+tar xvzf hypershift.tar.gz
```
Comment (rokej) [inline clusters/hosted_control_planes/hosted_install_cli.adoc]: hcp.tar.gz

File: clusters/hosted_control_planes/hosted_install_cli.adoc [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+----
+
+. Run the following command to make the binary file executable:
+
++
+----
+chmod +x hypershift
```
Comment (rokej) [inline clusters/hosted_control_planes/hosted_install_cli.adoc]: chmod +x hcp

File: clusters/hosted_control_planes/hosted_install_cli.adoc [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+----
+
+. Run the following command to move the binary file to a directory in your path:
+
++
+----
+sudo mv hypershift /usr/local/bin/.
```
Comment (rokej) [inline clusters/hosted_control_planes/hosted_install_cli.adoc]: sudo mv hcp /usr/local/bin/.

File: clusters/hosted_control_planes/hosted_install_cli.adoc [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+----
+
+You can now use the `hypershift create cluster` command to create and manage hosted clusters. Use the following command to list the available parameters:
+
+----
+hypershift create cluster aws --help
```
Comment (rokej) [inline clusters/hosted_control_planes/hosted_install_cli.adoc]: hcp create cluster ...

File: clusters/hosted_control_planes/hosted_install_cli.adoc [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+----
+
+. Run the following command to move the binary file to a directory in your path:
+
++
+----
+sudo mv hypershift /usr/local/bin/.
+----
+
+You can now use the `hypershift create cluster` command to create and manage hosted clusters. Use the following command to list the available parameters:
```
Comment (rokej) [inline clusters/hosted_control_planes/hosted_install_cli.adoc]: hcp create cluster

File: clusters/hosted_control_planes/hosted_install_cli.adoc (hunk 2 above)
Comment (lahinson) [reply clusters/hosted_control_planes/hosted_install_cli.adoc]: fixed

File: clusters/hosted_control_planes/hosted_install_cli.adoc (hunk 3 above)
Comment (lahinson) [reply clusters/hosted_control_planes/hosted_install_cli.adoc]: fixed

File: clusters/hosted_control_planes/hosted_install_cli.adoc (hunk 4 above)
Comment (lahinson) [reply clusters/hosted_control_planes/hosted_install_cli.adoc]: fixed

File: clusters/hosted_control_planes/hosted_install_cli.adoc (hunk 6 above)
Comment (lahinson) [reply clusters/hosted_control_planes/hosted_install_cli.adoc]: fixed

File: clusters/hosted_control_planes/hosted_install_cli.adoc (hunk 8 above)
Comment (lahinson) [reply clusters/hosted_control_planes/hosted_install_cli.adoc]: fixed

File: clusters/hosted_control_planes/hosted_install_cli.adoc (hunk 7 above)
Comment (lahinson) [reply clusters/hosted_control_planes/hosted_install_cli.adoc]: fixed

File: clusters/hosted_control_planes/hosted_install_cli.adoc (hunk 5 above)
Comment (lahinson) [reply clusters/hosted_control_planes/hosted_install_cli.adoc]: fixed

File: clusters/hosted_control_planes/hosted_install_cli.adoc [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+. Run the following command to move the binary file to a directory in your path:
++
+----
+sudo mv hcp /usr/local/bin/.
+----
+
+You can now use the `hcp create cluster` command to create and manage hosted clusters. To list the available parameters, enter the following command:
+
+----
+hcp create cluster aws --help
```
Comment (rokej) [inline clusters/hosted_control_planes/hosted_install_cli.adoc]: hcp create cluster \<platform\> --help

where supported \<platform\> are `aws`, `agent` and `kubevirt`.

Comment (lahinson) [reply clusters/hosted_control_planes/hosted_install_cli.adoc]: Good catch! I'll update that.

**Review by rokej — APPROVED** [review]

**Review by swopebe — APPROVED** [review]
I just scanned, since I am sure this has been peer reviewed already in the past. Thanks so much!

**Pull request merged**